### PR TITLE
Rbta 99/volodymyr cherniak/bug report/comments/general

### PR DESCRIPTION
**Pull request by Volodymyr-Cherniak**

### CODE REVIEW

File: test/selectors/bug-report.js [hunk 1]
```diff
@@ -32,8 +32,5 @@ export default {
   addCommentsForm: '.form-control',
   buttonAddComments: '#add_comment',
   buttonDeleteComment: '.hover-icon',
-  buttonAssignedToMe: '#to_me',
-  firstBugReport: '#summary_1 span'
```
Comment (artsenius) [inline test/selectors/bug-report.js]: Why did you remove these two entries? Is there any reason?

Comment (Volodymyr-Cherniak) [reply test/selectors/bug-report.js]: they are superfluous

**Comment by artsenius** [comment]
1. Remove screenshots.
2. do commit and push without screenshots
3. add screenshots to .gitignore or pull changes from integration
4. do another commit and push

**Comment by artsenius** [comment]
and fix the conflict
